source/_integrations/switchbot_cloud.markdown

```
@@ -40,25 +40,160 @@ Please note, device names configured in the SwitchBot app are transferred into H
+- [K10+ Pro](https://www.switch-bot.com/products/switchbot-mini-robot-vacuum-k10-pro)
+- [S1](https://www.switchbot.jp/products/switchbot-robot-vacuum-cleaner?&variant=41850919420079)
+- [S1 Plus](https://www.switchbot.jp/products/switchbot-robot-vacuum-cleaner)
+
+## Supported functionality
+
+### Plugs and switches
+
+#### Bot
+
+Features:
+- acted as a Switch in `switchMode` and `customizeMode`, as a Button in `pressMode`
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)